DS/DS6/EmployeeSortingByName.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+public class EmployeeSortingByName implements Comparable<Object>{
+	String employeeID,nameOfEmployee,addressOfEmployee;
```

Comment:
When Class Name is Employee than obviously variable of this class will contain Employee information, So no need to add employee in each and every variable. you can just use name,id (employeeID) and address.